apps/console/src/features/connections/components/edit/forms/authenticators/email-otp-authenticator-form.tsx

```diff
@@ -381,8 +381,8 @@ export const EmailOTPAuthenticatorForm: FunctionComponent<EmailOTPAuthenticatorF
                 </Label>
             </Field.Input>
             <Field.Checkbox
-                ariaLabel="Use numeric characters for OTP"
-                name="EmailOTP_OtpRegex_UseNumericChars"
+                ariaLabel="Use alphanumeric characters for OTP"
+                name="EmailOTP_OtpRegex_UseAlphanumericChars"
                 label={
                     t("console:develop.features.authenticationProvider.forms.authenticatorSettings" +
                         ".emailOTP.useNumericChars.label")
```

Comment:
It'd be better to update the i18n keys as well.

```suggestion
                        ".emailOTP.usealphaNumericChars.label")
```

Reply:
Resolve with 41555a63bde603bb36b8c7b0c1acf48eea654946